src/test/groovy/jp/co/soramitsu/devops/tasks/TasksTest.groovy

```diff
@@ -34,9 +34,9 @@ class TasksTest extends Specification {
         hasTask(result, SoraTask.dockerPush)
         hasTask(result, SoraTask.dockerVersion)
 
-        and: "has no this tasks"
-        !hasTask(result, 'jacocoTestReport')
-        !hasTask(result, 'jacocoTestCoverageVerification')
+//        and: "has no this tasks"
+//        !hasTask(result, 'jacocoTestReport')  TODO
```

Comment:
created another task for future, please refer to that in the todo:
`TODO: RLN-46`